site.yml

```diff
@@ -70,6 +70,7 @@
   - { role: contiv_storage, run_as: worker }
 
 # netplugin-node hosts set up netmast/netplugin in a cluster
+# Set contiv_network_mode to aci to run node in ACI mode
```

Comment:
I think this comment is not needed as we have provided possible values for `contiv_network_mode` in the vars file